src/Three.Legacy.js

```diff
@@ -1794,19 +1794,29 @@ Object.defineProperties( WebGLShadowMap.prototype, {
 
 } );
 
+export function WebGLRenderTargetCube( width, height, options ) {
+
+	console.warn( 'THREE.WebGLRenderTargetCube has been renamed to WebGLCubeRenderTarget.' );
+	return new WebGLCubeRenderTarget( width, height, options );
+
+}
+
 //
+/*
+// commenting out for now. added in https://github.com/mrdoob/three.js/pull/15808.
+// todo: handle this legacy warning, or remove it.
```

Comment:
/ping @Mugen87 I wasn't sure how to handle this, since the class no longer exists...

Reply:
At least a warning will appear. I removed the code.